### DISCUSSION TITLE
Is there Mermaid visual diff tool available?

### DISCUSSION THREAD

**Discussion by hyperrecursive** [mermaid-js/mermaid#4047] [Q&A]
Hello!

Is there Mermaid visual diff tool available? Obviously, one can diff the code. But is there a visual diff tool available? Such as when you supply the code of two diagrams into it, it will higlight added, removed and updated parts in different colors? I have found [this](https://github.com/mermaid-js/mermaid/issues/2833) issue where a user does pixel-based tests, but it's both unstable and not quite what I am looking for.

Would appreciate your suggestions!

**Comment by sidharthv96** [comment]
We are using applitools in our tests.
